### ISSUE TITLE
Resizing beyond the image border messes the image on Firefox

### ISSUE THREAD

**Issue by GabiAxel** [alexk111/ngImgCrop#43]
Tested the example http://jsfiddle.net/alexk111/rw6q9/ with Firefox 34.0.5 on Mac. Drag to resize and continue to drag beyond the image border. The shadow and selection mark remain on the image and can't be cleared.
Other browsers I tested didn't have this problem.

**Comment by MusharrafSakran** [comment]
Yes exactly, -in firefox- it will shatter the image if you drag the re-size arrow beyond the image border and then leave it, like this:

![croppercapture 41](https://cloud.githubusercontent.com/assets/2781157/5956227/9e939b56-a7bc-11e4-9f32-997d668a65ed.png)
